### ISSUE TITLE
Tags filter on masson commands

### ISSUE THREAD

**Issue by Pierrotws** [adaltas/node-masson#17]
propose the option -t to filter using tag in addition to module and host

Ex:
```bash
bin/masson install -t role=master -t environment=dev
```

**Comment by wdavidw** [comment]
I know, I had the same idea, but first, we need to re-write constrains

**Comment by Pierrotws** [comment]
Also, constraints are not constraints (since the operator used for multiple tags is an OR, not an AND).
Must be renammed.

**Comment by wdavidw** [comment]
No, must be re-implemented with AND. Not a priority before masson 2
